filters.config.js

```diff
@@ -1,4 +1,55 @@
+const {
+  FilterByTypeOptions,
+  FilterByAccessOptions,
+  FILTER_VALUES
+} = require('src/@types/aquarius/SearchQuery')
+
 module.exports = {
-  filters: [],
+  filters: [
+    {
+      id: 'serviceType',
+      label: 'Service Type',
+      type: 'filterList',
+      options: [
+        { label: 'datasets', value: FilterByTypeOptions.Data },
+        { label: 'algorithms', value: FilterByTypeOptions.Algorithm },
+        { label: 'saas', value: FilterByTypeOptions.Saas }
+      ]
+    },
+    {
+      id: 'accessType',
+      label: 'Access Type',
+      type: 'filterList',
+      options: [
+        { label: 'download', value: FilterByAccessOptions.Download },
+        { label: 'compute', value: FilterByAccessOptions.Compute }
+      ]
+    },
```

Comment:
suggestion(blocking): to reduce complexity and avoid having to define filters differently depending on the `id` we should simply move these filters over to the new specs. This would then look something like the following:

```ts
{
  id: 'accessType',
  label: 'Access Type',
  type: 'filterList',
  options: [
    {
      label: 'Download',
      value: FilterByAccessOptions.Download,
      queryPath: 'services.type'
    },
    {
      label: 'Compute',
      value: FilterByAccessOptions.Compute,
      queryPath: 'services.type'
    }
    //...
  ]
}        
//...
```